dynamo/models/cube.js

```diff
@@ -162,16 +163,31 @@ const hydrate = async (cube) => {
   }
 
   cube.owner = await User.getById(cube.owner);
+  cube.collaborators = await User.batchGet((cube.collaborators || []).filter((id) => id));
   cube.image = getImageData(cube.imageName);
-
+  
   return cube;
 };
 
 const batchHydrate = async (cubes) => {
   const owners = await User.batchGet(cubes.map((cube) => cube.owner).filter((owner) => owner));
 
+  const collaboratorIds = cubes.reduce((set, cube) => {
+    return new Set([...(set || []), ...(cube.collaborators | []).filter((id) => id)]);
+  });
+
+  const collaborators = await User.batchGet([...collaboratorIds]);
+
+  let collaboratorsById = new Map();
+  for (let collaborator of collaborators) {
+    if (collaborator && collaborator.id) {
```

Comment:
```suggestion
    if (collaborator?.id) {
```

Reply:
Didn't know this was supported in node now! nice :)